test/path.js

```diff
@@ -24,6 +24,7 @@ describe('path', function() {
     eq(R.path([], obj), obj);
     eq(R.path(['a', 'e', 'f', 1], obj), 101);
     eq(R.path(['j', 0], obj), 'J');
+    eq(R.path(['j', -1], obj), 'J');
```

Comment:
A couple things here. The object at `j` only has a single item in it, so it's not as good of a test of whether the negative indices work.  I would prefer to have a separate test for this behavior rather than just adding a case to this test.